postgraas_server/management_resources.py

```diff
@@ -137,6 +137,12 @@ def post(self):
         if not args['db_pwd']:
             abort(400, msg='The password may not be empty.')
 
+        if args['db_username'] == "postgres" or  "postgres@" in args['db_username']:
```

Comment:
Wouldn't it be easier to just check for `starts_with`? I am pretty sure the `@` would not be legal in any other position but I find it more explicit.

```suggestion
        if args['db_username'] == "postgres" or args['db_username'].startswith("postgres@"):
```

Also, I am wondering wether we do, and even should, allow `@` at all.

Reply:
Good idea, it also doesn't conflict with the other tests, which uses names like test_create_postgres_instance.

Reply:
* I suggest to be cautious here. IMHO `postgres_user` **should** be a valid value.
* https://www.postgresql.org/docs/11/auth-username-maps.html suggests that `@` by definition are valid in usernames.

Reply:
The former is why I only suggested that for the second part of the check.

I am aware that `@` is valid in usernames. This is required to have users that are host-specific. In our case we want to have users in a specific value for the host, though. Thus, it does not really make sense for the user to specify it.

Reply:
Ah, then I misunderstood you. Yeah, for `postgres@` your suggestion makes sense :).